.github/workflows/windows.yml

```diff
@@ -1,6 +1,6 @@
 name: windows
 
-on: [push]
+on: [push, pull_request]
```

Comment:
great, but why I dont see CI build status in the PR now? maybe you didn't enable GithubActionsCI in your fork?

Reply:
nope it's enabled